src/tests/Microsoft.PowerFx.Performance.Tests/Reference.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Diagnostics.Windows.Configs;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Jobs;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Performance.Tests
+{
+    [MemoryDiagnoser]
+    [NativeMemoryProfiler]    
+    [EtwProfiler] // https://benchmarkdotnet.org/articles/features/etwprofiler.html
+    [CsvExporter] // https://benchmarkdotnet.org/articles/configs/exporters.html
+    [MinColumn]
+    [Q1Column]
+    [MeanColumn]
+    [MedianColumn]
+    [Q3Column]
+    [MaxColumn]
+    [SimpleJob(runtimeMoniker: RuntimeMoniker.NetCoreApp31)]
+    public class Reference
+    {
+        [GlobalSetup]
+        public void GlobalSetup()
+        {
+            CultureInfo.CurrentCulture = new CultureInfo("en-US");
+        }
+
+        [Benchmark]
+        [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
+        public int FixedDuration()
+        {
+            // Fixed duration, independant of CPU speed 
+            Thread.Sleep(100);
+            return 0;
+        }
+
+        [Benchmark]
+        [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
+        public long Loop()
```

Comment:
Can we rename to "FixedLoop".

And add a comment in this class that the goal for all these tests is that they should take constant time (independent of any Power Fx changes), and so act as a baseline reference for performance.

Reply:
Done